client/src/layout/Drawer.jsx

```diff
@@ -0,0 +1,203 @@
+import AppBar from '@material-ui/core/AppBar';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Drawer from '@material-ui/core/Drawer';
+import Hidden from '@material-ui/core/Hidden';
+import IconButton from '@material-ui/core/IconButton';
+import { makeStyles, useTheme } from '@material-ui/core/styles';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import MenuIcon from '@material-ui/icons/Menu';
+import React, { useEffect, useState } from 'react';
+import Chatbox from '../components/ChatBox';
+import Users from '../components/Users';
+import Dropdown from '../layout/Dropdown';
+import { useVerify } from '../services/authenticationService';
+
+const drawerWidth = 342;
+
+const useStyles = makeStyles((theme) => ({
+	root: {
+		'& .MuiDrawer-paperAnchorDockedLeft': {
+			borderRight: 'none'
+		},
+		'& .MuiOutlinedInput-notchedOutline': {
+			borderColor: 'white'
+		},
+		display: 'flex',
+		borderRight: 'none'
+	},
+	drawer: {
+		[theme.breakpoints.up('sm')]: {
+			width: drawerWidth,
+			flexShrink: 0
+		}
+	},
+	dots: {
+		color: '#95A7C4'
+	},
+	onlineIcon: {
+		backgroundColor: '#1CED84',
+		fontSize: 13,
+		marginRight: 5,
+		width: 8,
+		height: 8,
+		borderRadius: 4
+	},
+	appBar: {
+		height: 89,
+		display: 'flex',
+		justifyContent: 'center',
+		backgroundColor: 'white',
+		color: 'black',
+		filter: 'drop-shadow(0px 2px 10px rgba(88,133,196,0.1))',
+		boxShadow: 'none',
+		[theme.breakpoints.up('sm')]: {
+			width: `calc(100% - ${drawerWidth + 25}px)`,
+			marginLeft: drawerWidth
+		}
+	},
+	menuButton: {
+		marginRight: theme.spacing(2),
+		[theme.breakpoints.up('sm')]: {
+			display: 'none'
+		}
+	},
+	// necessary for content to be below app bar
+	toolbar: theme.mixins.toolbar,
+	toolbar: {
+		display: 'flex',
+		justifyContent: 'space-between',
+		marginRight: 19
+	},
+	drawerPaper: {
+		width: drawerWidth
+	},
+	content: {
+		flexGrow: 1
+	},
+	onlineLabel: {
+		color: '#BFC9DB'
+	},
+	topBarContent: {
+		display: 'flex',
+		alignItems: 'baseline',
+		marginLeft: 10
+	}
+}));
+
+function ResponsiveDrawer(props) {
+	const { window } = props;
+	const classes = useStyles();
+	const theme = useTheme();
+
+	const [ mobileOpen, setMobileOpen ] = useState(false);
+	const [ onlineUsers, setOnlineUsers ] = useState({});
+	const [ currentUser, setCurrentUser ] = useState(null);
```

Comment:
I feel this would have been more logic to put in the Chat component (or a more parent level component). Also, I noticed the name of `user` as the other selected user could have been clearer